tests/common/widgets/current-refinements/options.ts

```diff
@@ -0,0 +1,406 @@
+import { createAlgoliaSearchClient } from '@instantsearch/mocks';
+import {
+  normalizeSnapshot as commonNormalizeSnapshot,
+  wait,
+} from '@instantsearch/testutils';
+import { queryByText } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+import type { CurrentRefinementsWidgetSetup } from '.';
+import type { TestOptions } from '../../common';
+
+function normalizeSnapshot(html: string) {
+  // React InstantSearch adds a noRefinement modifier on the list.
+  // @MAJOR: Remove this once React InstantSearch aligns with spec.
+  return commonNormalizeSnapshot(html).replace(
+    / ais-CurrentRefinements-list--noRefinement/g,
+    ''
+  );
```

Comment:
I think it would be a breaking change, removing this additional class name from React InstantSearch, so I added it to the snapshot normalization process.

Reply:
could we also add the --noRefinement class in the other flavours instead? or do you think it's useless (it's not very useful with :empty selector)

Reply:
I thought about it, but same as you, I really can't see any use for this. I'd rather wait and remove it next major.